### PR TITLE
Add projects and project sets

### LINKED ISSUE #4: Add projects model

### LINKED ISSUE #5: Add teams model

### PR DESCRIPTION
**Pull request by justino599**
Closes #4

### CODE REVIEW

File: app/models/projects.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
+from django.db import models
+
+from app.models.base_models import TimeStampedModel
+from app.models.course import Course
+
+
+class Project(TimeStampedModel):
+    name = models.CharField(max_length=100)
+    description = models.CharField(max_length=500, blank=True)
+    project_set_fk = models.ForeignKey(
```
Comment (Opeyem1a) [inline app/models/projects.py:10]: all things with `_fk` shouldn't need to have the fk, but we'll solve that on the base branch

**Review by Opeyem1a — CHANGES_REQUESTED** [review]

File: app/models/projects.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
+from django.db import models
+
+from app.models.base_models import BaseModel
+from app.models.course import Course
+
+
+class Project(BaseModel):
+    name = models.CharField(max_length=100)
+    description = models.CharField(max_length=500, blank=True)
+    project = models.ForeignKey(
```
Comment (Opeyem1a) [inline app/models/projects.py:10]:
```suggestion
    project_set = models.ForeignKey(
```

**Review by Opeyem1a — CHANGES_REQUESTED** [review]

File: app/models/projects.py (hunk 2 above)
Comment (justino599) [reply app/models/projects.py:10]: Good catch

**Comment by Opeyem1a** [comment]
Also deprecated by https://github.com/Teamable-Analytics/teamable-backend/pull/14

**Pull request closed without merging**